src/index.html

```diff
@@ -13,6 +13,52 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__top">
+        <img
+          src="images/imac.jpeg"
+          class="card__top-img"
+          alt="Apptle Monoblock"
+        />
+      </div>
+
+      <div class="card__bottom">
+        <h2 class="card__bottom-tittle">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__botoom-code">Product code: 195434</p>
+
+        <div class="card__botoom-stars">
```

Comment:
There is a typo in the class name `card__botoom-stars`. It should be `card__bottom-stars` to follow the BEM naming convention.